lib/buildpacks-ci-pipeline-update-command.rb

```diff
@@ -6,6 +6,12 @@ def run!(concourse_target_name:, pipeline_name:, config_generation_command:, pip
     buildpacks_configuration = BuildpacksCIConfiguration.new
 
     pipeline_prefix = ENV.fetch('PIPELINE_PREFIX', '')
+    secrets_file = ENV.fetch('CONCOURSE_SECRETS_FILE', nil)
+
+    if secrets_file.nil?
+      puts 'Please set CONCOURSE_SECRETS_FILE to the secrets location'
+      return false
```

Comment:
Not sure what this does, since it appears to just be called from [the bin script](https://github.com/SUSE/cf-buildpacks-ci/blob/12977757cc8c1522684067670c770000d7bf7425/bin/update-pipelines#L6) which then does nothing with the result.  Did you mean to `exit 1` or something like that?